src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.CustomCheckDetails.approved.txt

```diff
@@ -5,4 +5,5 @@ ServiceControl Health: Error Database Index Errors
 ServiceControl Health: Error Database Index Lag
 ServiceControl Health: Error Message Ingestion
 ServiceControl Health: Error Message Ingestion Process
+ServiceControl Health: Saga audit data retention check
```

Comment:
I would take the word "check" out. These are all checks

Reply:
And maybe make in capital letters as well?